packages/types/src/workflow/price-list/update-price-list.ts

```diff
@@ -0,0 +1,25 @@
+import { CreatePriceListRules, PriceListStatus } from "../../pricing"
```

Comment:
Added a ticket here to address this since this PR is really big already - https://linear.app/medusajs/issue/CORE-1572/workflow-types-are-moved-to-a-common-types

Reply:
The way we develop workflows right now is deeply linked to the core. But I can imagine this diverging in the future. My first thoughts is that workflows should really have its own types, but this might be overengineering at this point.